### ISSUE TITLE
Shell command available although its removed from the config

### ISSUE THREAD

**Issue by hrvstr** [vmikk/tshbot#2]
Hi, thanks for the awesome software. I tried setting up the bot, but it seems the `/shell` command still works even after I removed it from the config file and restarted the service. Any ideas on what I might be missing?

```
dietpi@DietPi:~$ cat .config/tshbot/tshbot.config
bot_log_file: "/var/log/tshbot.log"
bash_cmd: "/bin/bash"
tg_bot_token: "xyz"
tg_bot_chat_id: "abc"
allowed_cmds:
  pingg: 'ping -c 3 8.8.8.8'
  uptime: 'uptime'
  runscript: 'bash ~/bin/myscript.sh'
help_message: "Use /commands to see available commands."
```

```
dietpi@DietPi:~/.config/tshbot$ sudo systemctl stop tshbot.service
dietpi@DietPi:~/.config/tshbot$ sudo systemctl start tshbot.service
```

Running `/shell echo test` returns `test`

Edit:

Running `/commands` doesn't show the `/shell` command, even though it's still working.

![Bildschirmfoto 2024-12-16 um 14 34 12](https://github.com/user-attachments/assets/f42a37b7-c365-49fd-98d9-2a75f4ff1128)

**Comment by vmikk** [comment]
@hrvstr Hello Richard,
thank you for reporting this bug!
It has been fixed in the [newer release (v0.2)](https://github.com/vmikk/tshbot/releases/tag/0.2).  
Also, please note that there is now an additional parameter in the configuration file (`command_timeout`), which specifies the time (in seconds) after which old commands should be ignored.

**Comment by hrvstr** [comment]
Thanks for the quick fix. Everything's working fine now!
